casper/src/main/scala/coop/rchain/casper/genesis/Genesis.scala

```diff
@@ -62,13 +65,13 @@ object Genesis {
       StandardDeploys.testRev(genesisPk, vaults, supply),
       StandardDeploys.testPoS(posParams)
     ) ++ posParams.validators.map { validator =>
-    DeployData(
+      DeployData(
       deployer = ByteString.copyFrom(validator.pk.bytes),
       timestamp = System.currentTimeMillis(),
       term = validator.code,
       phloLimit = accounting.MAX_VALUE
     )
-  }
+    }
```

Comment:
Are there potentially buggy features in EAP?

Reply:
In fact, the latest 2019.1 had a pref regression, but the EAP fixed that pretty fast :) I'm only recommending EAP because I've upgraded to it recently and it seems to work OK

Reply:
The latest is 2019.1.1 RC, so should be pretty stable

Reply:
I've tried to upgrade it, but after the upgrade IntelliJ refuses to start. I'll upgrade and troubleshoot, but I don't see why that should block this PR.